### DISCUSSION TITLE
Skip building AppImage when compiling on Linux

### DISCUSSION THREAD

**Discussion by el-myz** [tauri-apps/tauri#7867] [Q&A] [answered]
Hi,

I wonder if there is a way to configure Tauri not to build an AppImage bundle of my application? 
I could not find the correct configuration object for this.

Best regards

**Comment by FabianLars** [comment, marked as answer]
You're looking for bundle.targets in tauri.conf.json: <https://tauri.app/v1/api/config#bundleconfig.targets> (it's typically set to `"all"`).

For a more temporary approach you can also use the --bundle flag, for example `tauri build --bundle deb` for a .deb file or `tauri build --bundle none` for just the binary and no bundle.

**Reply by el-myz** [reply]
Thank you very much! Now that you say it, I see it is even mentioned in the guide in the flaming red box. Sorry to bother.

**Reply by creeper12356** [reply]
Notice: til now the `--bundle` option has been changed to `--bundles`. :smile:
